src/chainstate/stacks/boot/sns.clar

```diff
@@ -0,0 +1,970 @@
+;;;; Errors
+(define-constant err-panic 0)
+(define-constant err-namespace-preorder-not-found 1001)
+(define-constant err-namespace-preorder-expired 1002)
+(define-constant err-namespace-preorder-already-exists 1003)
+(define-constant err-namespace-unavailable 1004)
+(define-constant err-namespace-not-found 1005)
+(define-constant err-namespace-already-exists 1006)
+(define-constant err-namespace-not-launched 1007)
+(define-constant err-namespace-price-function-invalid 1008)
+(define-constant err-namespace-preorder-claimability-expired 1009)
+(define-constant err-namespace-launchability-expired 1010)
+(define-constant err-namespace-operation-unauthorized 1011)
+(define-constant err-namespace-stx-burnt-insufficient 1012)
+(define-constant err-namespace-blank 1013)
+(define-constant err-namespace-already-launched 1014)
+(define-constant err-namespace-hash-malformed 1015)
+(define-constant err-namespace-charset-invalid 1016)
+
+(define-constant err-name-preorder-not-found 2001)
+(define-constant err-name-preorder-expired 2002)
+(define-constant err-name-preorder-funds-insufficient 2003)
+(define-constant err-name-unavailable 2004)
+(define-constant err-name-operation-unauthorized 2006)
+(define-constant err-name-stx-burnt-insufficient 2007)
+(define-constant err-name-expired 2008)
+(define-constant err-name-grace-period 2009)
+(define-constant err-name-blank 2010)
+(define-constant err-name-already-claimed 2011)
+(define-constant err-name-claimability-expired 2012)
+(define-constant err-name-not-found 2013)
+(define-constant err-name-revoked 2014)
+(define-constant err-name-transfer-failed 2015)
+(define-constant err-name-preorder-already-exists 2016)
+(define-constant err-name-hash-malformed 2017)
+(define-constant err-name-preordered-before-namespace-launch 2018)
+(define-constant err-name-not-resolvable 2019)
+(define-constant err-name-could-not-be-minted 2020)
+(define-constant err-name-could-not-be-transfered 2021)
+(define-constant err-name-charset-invalid 2022)
+
+(define-constant err-principal-already-associated 3001)
+(define-constant err-insufficient-funds 4001)
+(define-constant err-zonefile-not-found 5001)
+
+;;;; Constants
+(define-constant burn-address 'S0000000000000000000002AA028H)
```

Comment:
does this address work for both testnet and mainnet, or is this expected to only work on mainnet only?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/0bdf51f9f85cb45ddf8340b45838c7d5ef1c6984